plugins/git/functions

```diff
@@ -60,7 +60,7 @@ git_build_app_repo() {
   suppress_output git submodule update --init --recursive
   find . -name .git -prune -exec rm -rf {} \; > /dev/null
 
-  plugn trigger post-extract "$APP" "$GIT_BUILD_APP_REPO_TMP_WORK_DIR" "$REV"
+  plugn trigger post-extract "$APP" "$GIT_BUILD_APP_REPO_TMP_WORK_DIR" "$REV" | sed -u "s/^/"$'\e[1G'"/"
```

Comment:
This removes the `remote:` prefix that we normally remove when it's just triggered by `pre-receive-app`.